bundles/binding/org.openhab.binding.fatekplc/src/main/java/org/openhab/binding/fatekplc/items/FatekContactItem.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.fatekplc.items;
+
+import java.util.List;
+import java.util.Map;
+
+import org.openhab.core.items.Item;
+import org.openhab.core.library.types.OpenClosedType;
+import org.openhab.core.types.State;
+import org.openhab.model.item.binding.BindingConfigParseException;
+
+import com.github.s4u.jfatek.registers.Reg;
+import com.github.s4u.jfatek.registers.RegValue;
+
+/**
+ * Contact item implementation.
+ *
+ * @author Slawomir Jaranowski
+ * @since 1.9.0
+ */
+public class FatekContactItem extends FatekContacSwitchtItem {
```

Comment:
Class `FateContactSwitchItem` was implemented as common code for Contact and Switch item.
Now I see that maybe this class is not necessary.